extra-shells/elvish/autobuild/postrm

```diff
@@ -0,0 +1,8 @@
+# This command is adopted from Arch Linux's elvish.install
+# Reference: 
+# https://github.com/archlinux/svntogit-community/blob/packages/elvish/trunk/elvish.install
+if [ $1 = 'remove' ]; then
+    echo "Attempting to purge elvish from /etc/shells ..."
```

Comment:
Again, no need to add outputs like this. We might address this in a shell survey for consistency.